Rubberduck.Core/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -232,6 +234,106 @@ public bool GroupByLocation
             }
         }
 
+        private List<CodeInspectionSeverity> _filterCriteria = new List<CodeInspectionSeverity>();
+
+        public bool FilterInspectionsByHint
+        {
+            get => _filterCriteria.Contains(CodeInspectionSeverity.Hint);
+            set
+            {
+                if (FilterInspectionsByHint == value) { return; }
```

Comment:
From a purely code style standpoint, I'm not sure how I feel about calling the getter from the setter. This may be moot though, in that it can go away if a method is extracted.